ciphey/ciphey.py

```diff
@@ -18,9 +18,10 @@
 import click
 from appdirs import AppDirs
 from loguru import logger
-from rich import print
 from yaspin import yaspin
 from yaspin.spinners import Spinners
+from rich.console import Console
+console = Console()
```

Comment:
Note: I do this a few times instead of making one rich console object and passing it through the config. Will make an issue :-)